src/game/screen.h

```diff
@@ -11,13 +11,16 @@ int32_t Screen_GetResIdx(void);
 int32_t Screen_GetResWidth(void);
 int32_t Screen_GetResHeight(void);
 int32_t Screen_GetResWidthDownscaled(void);
+int32_t Screen_GetResWidthDownscaledBar(void);
```

Comment:
If you agree with my other suggestion we should rename `Screen_GetResWidthDownscaled` to `Screen_GetResWidthDownscaledText` as well for consistency.